scripts/publish_pr.py

```
@@ -1,423 +1,18 @@
-        if body_file:
-            body_file.unlink(missing_ok=True)
-
-
-def check_gh_installed() -> bool:
-    """Check if gh CLI is installed and accessible."""
-    try:
-        subprocess.run(
-            ["gh", "--version"],
+if __name__ == "__main__":
+    # Forward all arguments to the new location
+    result = subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)